### ISSUE TITLE
[Feature Request]: Create a Contact us Page.

### ISSUE THREAD

**Issue by Pratik2026** [bsoc-bitbyte/busify#100]
### Is your feature request related to a problem? Please describe.

We need a contact page so that users can get the details required to communicate with the team.

### Describe the solution you'd like

The Contact Us page should contain three cards showcasing the help center, email, and phone details. Refer to the image for the structure and content:

![image](https://github.com/bsoc-bitbyte/busify/assets/128511266/15cd190d-1826-4640-a0d1-30b75954b83a)

The design should be such that at the top, there should be a cover image. Below the cover image, these three cards will be displayed in a row. It would be great if the cards would overlap the cover image by 20% from the bottom of the image. You can decide on the color scheme on your own.

Creativity is highly appreciated :)

**Comment by AnupamSingh2004** [comment]
@Pratik2026 I would like to work on this issue and create a Contact Us page for the website

**Comment by Pratik2026** [comment]
@AnupamSingh2004, Can you tell me your progress on this issue? It's been 3 days since the issue has been assigned to you.

**Comment by AnupamSingh2004** [comment]
Yes sir working on the the contact us page, can I know what the cover image shoul be on the page or should I leave it blank for anyone to put it later.

**Comment by Pratik2026** [comment]
Let's keep a random cover image for now. You can DM me on Discord for design-related discussion @AnupamSingh2004
